cfgov/unprocessed/js/modules/YoutubePlayer.js

```diff
@@ -0,0 +1,84 @@
+/* ==========================================================================
+   YouTube Initialization
+   ========================================================================== */
+
+'use strict';
+
+var _noopFunct = require( './util/standard-type' ).noopFunct;
+var _noopFunct = require( './util/standard-type' ).noopFunct;
+var VideoPlayer = require( './VideoPlayer' );
+
+var YoutubePlayer = VideoPlayer.extend( {
+
+  constructor: YoutubePlayer,
+
+  SCRIPT_API: 'http://www.youtube.com/iframe_api',
+
+  iFrameProperties: {
+    id: 'video-player_iframe__youtube'
+  },
+
+  playerOptions: {
+    videoId: '',
+    playerVars: {
+      autoplay: 1,
+      suggestedQuality: 'highres'
+    },
+    events: {
+      onReady: _noopFunct,
+      onStateChange: _noopFunct
+    }
+  },
+
+  playerInitialized: false,
+
+  embedScript: function( ) {
+    window.onYouTubeIframeAPIReady = this.initPlayer;
+    this._super.embedScript.call(  this.SCRIPT_API );
```

Comment:
If this is set on the prototype, what about making it a static method on the constructor?
`VideoPlayer.embedScript.call( YoutubePlayer.SCRIPT_API );`